dictionary.txt

```diff
@@ -0,0 +1,656 @@
+250kb
```

Comment:
man this is a huge dictionary :( any way to set ignores for the cli? for example we can just ignore sections with contributors to not add all of them here in the dict

Reply:
Yeah this is possible! The tool allows for programmatic dictionaries: JavaScript files that export lists of words to be ignored. I could write a programmatic dictionary that parses the frontmatter in each Markdown file and generates a list of all contributors' names to be ignored. The end result is that new contributors wouldn't have to add their name to the dictionary - it would be automatically ignored once added to the frontmatter of any Markdown file.

Reply:
That would be unnecessary work, it is better to avoid adding them, avoid adding a new file and avoid just doing it.

I am talking about this API: https://github.com/tbroadley/spellchecker-cli/blob/master/index.js#L43, your package doesn't export it, but it is there.

As you said, parse file's frontmatter, and only run the spellchecker in the title and content, of course, a dictionary may be necessary for all the tech/webpack related terms, but it would be significantly smaller and we won't have to update on almost every PR.

Reply:
That makes sense! I'd be happy to add functionality to the tool to allow spell-checking only specific tags in the frontmatter.

Would you mind explaining why you think it would be better to use the internal API directly, rather than the CLI wrapping it?

Reply:
As my previous comment:
>  only run the spellchecker in the title and content

CLI is checking the entire file.